packages/flutter_tools/lib/src/localizations/gen_l10n_types.dart

```diff
@@ -345,6 +345,7 @@ class Message {
       value = _value(templateBundle.resources, resourceId),
       description = _description(templateBundle.resources, resourceId, isResourceAttributeRequired),
       placeholders = _placeholders(templateBundle.resources, resourceId, isResourceAttributeRequired),
```

Comment:
should we rename `placeholders` to `templatePlaceholders` to better differentiate between it and `localePlaceholders`?

Reply:
I renamed it.